### PR TITLE
fix the awc/gcp bug

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/views/SslList.vue [hunk 1]
```diff
@@ -274,8 +274,11 @@ export default defineComponent({
         {
           title: 'AWS',
           fieldNames: ['links'],
-          displayFunction: (item) => {
-            return item?.links?.provider === 'aws' ? 'true' : 'false'
+          displayFunction: (item: Certificate) => {
+            const isAWS = _.some(item?.links, (link: Link) => {
+              return link?.provider === 'aws' ? true : false
```
Comment (Aviv-Galmidi) [inline src/views/SslList.vue:279]: return statement can be simplified

```suggestion
              return link?.provider === 'aws'
```

File: src/views/SslList.vue [hunk 2]
```diff
@@ -285,8 +288,11 @@ export default defineComponent({
         {
           title: 'GCP',
           fieldNames: ['links'],
-          displayFunction: (item) => {
-            return item?.links?.provider === 'gcp' ? 'true' : 'false'
+          displayFunction: (item: Certificate) => {
+            const isGCP = _.some(item?.links, (link: Link) => {
+              return link?.provider === 'gcp' ? true : false
```
Comment (Aviv-Galmidi) [inline src/views/SslList.vue:293]: return statement can be simplified

```suggestion
              return link?.provider === 'gcp'
```

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

**Review by Aviv-Galmidi — APPROVED** [review]

File: src/views/SslList.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/views/SslList.vue]: Done

File: src/views/SslList.vue (hunk 2 above)
Comment (AvihayMReblaze) [reply src/views/SslList.vue]: Done

**Pull request merged**